packer/builder/azure/driver_powershell/builder.go

```diff
@@ -80,6 +83,8 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		"location":                  &b.config.Location,
 		"instance_size":             &b.config.InstanceSize,
 		"user_image_label":          &b.config.UserImageLabel,
+		"vnet":                      &b.config.VNet,
+		"subnet":                    &b.config.Subnet,
```

Comment:
Yea i didnt care about the powershell version but i will add it. Sorry bout that just because i dont care doesnt mean others dont either ;)